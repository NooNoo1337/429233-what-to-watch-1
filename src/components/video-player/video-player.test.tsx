import * as React from 'react';
import * as renderer from 'react-test-renderer';

import VideoPlayer from './video-player';
const mockVideo = [
  {
    'preview_image': `img/the-grand-budapest-hotel.jpg`,
    'preview_video_link': `https://some-link`,
  }
];

describe(`GenresListComponent`, () => {
  it(`should render component correctly`, () => {
    const tree = renderer
      .create(<VideoPlayer
        posterSrc={mockVideo[`preview_image`]}
        videoSrc={mockVideo[`preview_video_link`]}
        videoFormat="video/mp4"
        isVideoPlaying={false}
        isSoundOff={true}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
