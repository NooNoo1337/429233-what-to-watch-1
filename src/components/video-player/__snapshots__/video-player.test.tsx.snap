// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresListComponent should render component correctly 1`] = `
<video
  className="small-movie-card__image"
  height="175"
  muted={true}
  width="280"
>
  <source
    type="video/mp4"
  />
</video>
`;
